src/main/kotlin/com/rstudio/shinycannon/Main.kt

```diff
@@ -364,13 +394,37 @@ class EnduranceTest(val argsStr: String,
 
 }
 
+fun parseHeader(header: String): Header {
+    val parts = header.split(":")
+    check(parts.size == 2) { "Header is malformed" }
+    val (name, value) = parts
+    check(name.isNotEmpty()) { "Header name is empty" }
+    return BasicHeader(name, value.replace("^\\s+", ""))
+}
+
+fun parseKey(key: String?): Header? {
+    if (key == null) return null
+    if (key.length == 0) return null
+
+    return BasicHeader("Authorization", "Key ${key}")
+}
+
 class Args(parser: ArgParser) {
     val recordingPath by parser.positional("Path to recording file")
     val appUrl by parser.positional("URL of the Shiny application to interact with")
     val workers by parser.storing("Number of workers to simulate. Default is 1.") { toInt() }
             .default(1)
     val loadedDurationMinutes by parser.storing("Number of minutes to continue simulating sessions in each worker after all workers have completed one session. Can be fractional. Default is 5.") { toBigDecimal() }
             .default(BigDecimal(5))
+    val startInterval by parser.storing("Number of milliseconds to wait between starting workers. Defaults to the length of the recording divided by the number of workers.") {
+        toLong()
+    }.default(null)
+    val headers by parser.adding("-H", "--header", help = "A custom HTTP header in the form 'name: value' to add to each request.") {
+        parseHeader(this)
+    }
+    val connectApiKey by parser.storing("-K", "--connect-api-key", help = "An RStudio Connect API key.") {
+        parseKey(this)
+    }.default(parseKey(System.getenv("SHINYCANNON_CONNECT_API_KEY")))
```

Comment:
`--connect-api-key` option should be removed--only the environment variable should be respected.